src/app/trainer/dashboard/clients/page.jsx

```
@@ -1,57 +1,52 @@
+  const [totalClients, setTotalClients] = useState(0);
+  const [sortOption, setSortOption] = useState("name");
+  const [sortOrder, setSortOrder] = useState("asc");
+  const searchInputRef = useRef(null);
+
+  // Pagination settings
+  const CLIENTS_PER_PAGE = 12;
 
   // Get tracking parameters
   const planId = searchParams.get("planId");
   const planType = searchParams.get("type");
+  const planId = searchParams.get("planId");
```

Biome (2.1.2)

[error] Shouldn't redeclare 'planId'. Consider to delete it or rename it.

'planId' is defined here:


(lint/suspicious/noRedeclare)